ONSdigital/response-operations-ui
# Find case by postcode

# Motivation and Context
We're adding functionality to the social end of the response operations UI to enable the user to search for cases by post code.

# What has changed
* New controllers are added to communicate with the sample and case services to retrieve cases relating to a post code
* A new social page has been added to search by postcode and display results
* Refactor to the base html templates, factored common patterns out into partials and added new a new social base

# How to test?
* Run the social test setup in rm-tools to get some social cases
* After a couple of minutes (for the cases to be created), try searching for postcodes in your sample file, you should see results 
* Check acceptance tests pass

# Links
[Sample service PR](https://github.com/ONSdigital/rm-sample-service/pull/62)
https://trello.com/c/uqLRhZiZ/189-sus002-find-case-by-postcode-13
